tests/alerts_tests.py

```diff
@@ -41,95 +50,204 @@
 def setup_database():
     with app.app_context():
         slice_id = db.session.query(Slice).all()[0].id
-        database_id = utils.get_example_database().id
+        database = utils.get_example_database()
+        database_id = database.id
+        database.get_sqla_engine().execute("CREATE TABLE test_table AS SELECT 2 as id")
 
+        common_data = dict(
+            active=True,
+            crontab="* * * * *",
+            slice_id=slice_id,
+            recipients="recipient1@superset.com",
+            slack_channel="#test_channel",
+        )
         alerts = [
-            Alert(
-                id=1,
-                label="alert_1",
-                active=True,
-                crontab="*/1 * * * *",
-                sql="SELECT 0",
-                alert_type="email",
-                slice_id=slice_id,
-                database_id=database_id,
+            Alert(**common_data, id=1, label="alert_1"),
```

Comment:
let's not specify ids here and those are autogenerated, use label to retrieve alerts you care about

Reply:
I specified id's in order to control with SQLObserver is linked to which alert (alert_id column in SQLObserver), I'm not sure if there is another way to do this